apps/design-system-docs/package.json

```diff
@@ -26,15 +26,19 @@
 		"@docusaurus/core": "^3.7.0",
 		"@docusaurus/preset-classic": "^3.7.0",
 		"@mdx-js/react": "^3.0.0",
+		"@wordpress/components": "^29.9.0",
+		"docusaurus-plugin-sass": "^0.2.6",
 		"prism-react-renderer": "^2.3.0",
 		"react": "^18.3.1",
-		"react-dom": "^18.3.1"
+		"react-dom": "^18.3.1",
+		"sass": "^1.54.0"
```

Comment:
Can we use [`sass-embedded`](https://www.npmjs.com/package/sass-embedded) instead, for _speed_? AFAIK it should ["just work"](https://github.com/webpack-contrib/sass-loader#:~:text=By%20default%2C%20the%20loader%20resolves%20the%20implementation%20based%20on%20your%20dependencies.%20Just%20add%20the%20desired%20implementation%20to%20your%20package.json%20(sass%2C%20sass%2Dembedded%2C%20or%20node%2Dsass%20package)%20and%20install%20dependencies.) with the underlying `sass-loader`.

Reply:
That worked, thanks! 7a1fda4

Reply:
Not in this PR, but some more possible optimizations: https://github.com/facebook/docusaurus/issues/10556